mmv1/templates/terraform/examples/backup_dr_management_server.tf.erb

```diff
@@ -0,0 +1,28 @@
+resource "google_compute_network" "default" {
+  name = "<%= ctx[:vars]['network_name'] %>"
+}
+
+resource "google_compute_global_address" "private_ip_address" {
+  name          = "<%= ctx[:vars]['network_name'] %>"
+  address_type  = "INTERNAL"
+  purpose       = "VPC_PEERING"
+  prefix_length = 20
+  network       = google_compute_network.default.id
+}
+
+resource "google_service_networking_connection" "default" {
+  network                 = google_compute_network.default.id
+  service                 = "servicenetworking.googleapis.com"
+  reserved_peering_ranges = [google_compute_global_address.private_ip_address.name]
+}
+
+resource "google_backup_dr_management_server" "<%= ctx[:primary_resource_id] %>" {
+  location = "us-central1"
+  name     = "<%= ctx[:primary_resource_id] %>"
```

Comment:
Can this be taken in as a variable name instead? 

Similar to `<%= ctx[:vars]['network_name'] %>`. That way we also get the randomized suffix

[modular-magician/terraform-provider-google-beta@`auto-pr-8568-old..auto-pr-8568`#diff-0a4e274e06](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-8568-old..auto-pr-8568#diff-0a4e274e065b3857c546850aa89f2d0922a21ef867a3296e199705cf84da4e22R67)